### PR TITLE
Move System.Deployment to msbuild-api-docs

### PR DESCRIPTION
**Pull request by v-alje**
Build.Reason:Manual by Alma Jenks
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=298390&view=results
 source_repo.branch:release-msbuild
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/FrameworksIndex/netcore-1.0.xml [hunk 1]
```diff
@@ -11240,16 +11240,6 @@
       <Member Id="F:System.IO.FileMode.OpenOrCreate" />
       <Member Id="F:System.IO.FileMode.Truncate" />
     </Type>
-    <Type Name="System.IO.FileNotFoundException" Id="T:System.IO.FileNotFoundException">
```
Comment (gewarren) [inline xml/FrameworksIndex/netcore-1.0.xml:11243]: I'm not sure why this is removed. According to apisof.net, this type is available in .NET Core 1.0 from the System.Runtime.dll assembly.

File: xml/FrameworksIndex/netstandard-1.5.xml [hunk 2]
```diff
@@ -3144,27 +3144,11 @@
       <Member Id="P:System.IO.FileLoadException.FileName" />
       <Member Id="P:System.IO.FileLoadException.Message" />
     </Type>
-    <Type Name="System.IO.FileNotFoundException" Id="T:System.IO.FileNotFoundException">
```
Comment (gewarren) [inline xml/FrameworksIndex/netstandard-1.5.xml:3147]: Also here.

File: xml/FrameworksIndex/netcore-1.0.xml (hunk 1 above)
Comment (v-alje) [reply xml/FrameworksIndex/netcore-1.0.xml:11243]: That is weird. I copied the configuration parameters from the other CI job, and the dll XML changes aren't touching .NET Core.

Comment (huangmin-ms) [reply xml/FrameworksIndex/netcore-1.0.xml:11243]: @gewarren I will take a look tomorrow.

Comment (huangmin-ms) [reply xml/FrameworksIndex/netcore-1.0.xml:11243]: @gewarren I was not able to reproduce this in my local. @v-alje would you mind opening a bug regarding this issue? Thanks!

Comment (v-alje) [reply xml/FrameworksIndex/netcore-1.0.xml:11243]: @huangmin-ms Sure. Bug has been opened: https://dev.azure.com/ceapex/Engineering/_workitems/edit/639800

**Pull request closed without merging**